### ISSUE TITLE
Pre-commit can't find nvm.sh

### ISSUE THREAD

**Issue by michalkvasnicak** [typicode/husky#144]
After update from `0.13.4` to `0.14.0` `pre-commit` hook stopped working with the following message:

```
.git/hooks/pre-commit: line 19: /Users/Michal/.nvm/nvm.sh: No such file or directory
```

I downgraded back to `0.13.4` and everything works fine.

**Comment by michalkvasnicak** [comment]
There is no `nvm.sh` in `/Users/Michal/.nvm/nvm.sh`. I tried to upgrade nvm using brew but this directory stays the same.

I have `nvm.sh` stored only in `/usr/local/opt/nvm`.

**Comment by alemhnan** [comment]
Same problem for me.

**Comment by typicode** [comment]
Thanks for letting me know, just published a fix in `v0.14.1` :)

**Comment by camsong** [comment]
it works,thank
